client/src/common/components/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Footer = () => {
+	return (
+		<footer className="flex justify-center items-center bg-header_color h-mobileHeaderHeight md:h-laptopHeaderHeight">
+			<p className="text-primary_bg_color text-center text-xs">
+				Copyright ©2021.{' '}
+				<Link
+					to="https://github.com/shiv-chan/senpai-kohai"
```

Comment:
Maybe we can just use a `a` tag here instead of `Link`? I couldn't find the exact information about this on the docs of React Router, but probably `Link` is not really for external links.
And I clicked the link but now the URL looks like below 😮 `http://localhost:3000/https://github.com/shiv-chan/senpai-kohai`

Reply:
I mixed up this with Next.js...😂
Yes, we wanna replace it with `<a>` tag.  Thank you for your review @miyabitanimchi !